### ISSUE TITLE
Console doesn't indicate input possibility (>_)

### ISSUE THREAD

**Issue by sphinxc0re** [cuberite/cuberite#1812]
The console should show something like > or $ at the start to indicate a possible input from the user. This should be a rather easy change

**Comment by madmaxoft** [comment]
Not quite, because the console is free to output more text at any time, which would interfere with any input prompt

**Comment by madmaxoft** [comment]
And, at least on Windows, you can write text at any time and the server will process it when it finishes any work being done (yes, you can actually write commands before the server is fully started)

**Comment by SphinxC0re** [comment]
I have a functioning solution for that. I'll submit a PR and you can see what you think

**Comment by SphinxC0re** [comment]
Has anyone else than me the opinon that we need such an indicator?

**Comment by tigerw** [comment]
It would be aesthetically pleasing, but likely challenging to reliably and portably develop.

**Comment by madmaxoft** [comment]
We might aim for some kind of a better UI by making the server more UI-less and then developing specific UIs for specific platforms - just like the vanilla server can use a console *or* it creates a nice window with a separate input for the console commands.

**Comment by SphinxC0re** [comment]
We could use Qt for the UI so we won't have to worry about cross-compatibility

**Comment by jan64** [comment]
Imo, adding window-based UI should wait until the server has capabilities almost identical with the vanilla ones. Also, adding any sort of windowed interface seems like too much of a hassle given that we  already have the webadmin.

**Comment by williamhatcher** [comment]
The console now indicates input possible. We can close this

**Comment by madmaxoft** [comment]
@williamhatcher it does? I haven't noticed anything.

**Comment by williamhatcher** [comment]
I misread what the OP was asking for, sorry.
